ccd-definition/CaseField/CareSupervision/caseField.json

```diff
@@ -636,6 +636,14 @@
     "FieldType": "FurtherDirections",
     "SecurityClassification": "Public"
   },
+  {
```

Comment:
More of a note to self to move this when we merge our stories, I have separated out all the case fields for the create order event into their own file so it will be easier to see what is used for the event.